routes/web.php

```diff
@@ -21,6 +23,11 @@
 // Admin
 
 Route::group(['prefix' => 'admin'], function () {
+    Route::get('login', 'Admins\AdminController@getLogin')->name('getLoginAdmin');
+    Route::post('login', 'Admins\AdminController@postLogin')->name('postLoginAdmin');
+        Route::get('logout', 'Admins\AdminController@getLogout')->name('getLogout');
+    });
```

Comment:
```suggestion
    Route::get('login', 'Admins\AdminController@getLogin')->name('admin.get_login');
    Route::post('login', 'Admins\AdminController@postLogin')->name('admin.post_login');
    Route::get('logout', 'Admins\AdminController@getLogout')->name('get_logout');
});
```
name route đặt snake case nhé anh

Reply:
oke em